add getStats test

KaptenJansson opened the pull request:
@fippo PTAL

fippo commented on test/test.js:
```diff
@@ -30,3 +30,35 @@ test('Check Selenium lib buildDriver method', function(t) {
+  .then(function() {
+    t.plan(3);
+    t.pass('Page loaded');
+    return driver.executeScript('window.pc1 = new RTCPeerConnection;' +
```
did you omit the () intentionally? Didn't even know that was possible.

KaptenJansson replied:
No that was a mistake when cleaning up :(

fippo commented:
needs skip-test for Firefox. LGTM otherwise.
Can you also turn off travis' build on push?

KaptenJansson commented:
Turned off build on push, addressed comments (oddly enough `throw 'skip test`' did not work). Also replaced jshint env comments with eslint env. And updated .gitignore + added copyright.

fippo commented:
:+1:
